### ISSUE TITLE
Run to line

### ISSUE THREAD

**Issue by tsmaeder** [eclipse-theia/theia#3424]
Implement an action that runs to a line in the currently selected frame. Will fill in the details in this issue, soon.

**Comment by colin-grant-work** [comment]
@tsmaeder, are you still interested in this feature / is it still missing?

**Comment by pisv** [comment]
@colin-grant-work Can you please close this issue? I think it has been addressed in #16219, but was not closed automatically.

**Comment by colin-grant-work** [comment]
Closed by https://github.com/eclipse-theia/theia/pull/16219

**Comment by colin-grant-work** [comment]
@pisv, yes, unfortunately, you have to use one of the magic words before every issue `closes #..., closes #...`; GitHub doesn't understand `closes #..., #...`.

**Comment by pisv** [comment]
@colin-grant-work Oh, good to know! Thank you 👍
